analyzer/src/main/kotlin/managers/Unmanaged.kt

```diff
@@ -64,6 +64,6 @@ class Unmanaged : PackageManager() {
                 scopes = sortedSetOf()
         )
 
-        return ProjectAnalyzerResult(Main.allowDynamicVersions, project, sortedSetOf())
+        return ProjectAnalyzerResult(config, project, sortedSetOf())
```

Comment:
Arguably the `Unmanaged` "package manager" also does not support lock files. So maybe this should be hard-coded here for consistency. However this would be a clear case of [Hobgoblin Consistency](https://en.wikipedia.org/wiki/Self-Reliance).

I'm actually more in favor of removing the hard-coding in the other cases as well. The `config` section in the analyzer result simply reflects the users/callers input, regardless of whether all those flags are applicable in every case. Of course we might not want to do that since it changes the semantics and not just the format, as this change currently does.

Reply:
You're right about what you're saying in the second paragraph. I've addressed this in a new preceding commit in this PR.